examples/commitsRUsPlaceTrackOrder.json

```
@@ -0,0 +1,1070 @@
+    },
+    {
+      "project_id":"b3fpk8",
+      "trace_type":"text",
+      "trace_payload":"{\"payload\":{\"message\":\"Let me track your order. What is your order number?\"}}"
+    },
+    {
+      "project_id":"b3fpk8",
+      "trace_type":"text",
+      "trace_payload":"{\"payload\":{\"message\":\"We have found your order for Add Chatbot pull request. What would you like to know?\"}}"
+    },
+    {
+      "project_id":"b3fpk8",
+      "trace_type":"text",
+      "trace_payload":"{\"payload\":{\"message\":\"It is expected to arrive on November 8th, at 10 am.\"}}"
+    },
+    {},
+    {
+      "project_id":"b3fpk8",
+      "trace_type":"text",
+      "trace_payload":"{\"payload\":{\"message\":\"Welcome to Commits 'R Us, the best toy store in town. How can I help you?\"}}"
+    },
+    {
+      "project_id":"b3fpk8",
+      "trace_type":"text",
+      "trace_payload":"{\"payload\":{\"message\":\"Let me track your order. What is your order number?\"}}"
+    },
+    {
+      "project_id":"b3fpk8",
+      "trace_type":"text",
+      "trace_payload":"{\"payload\":{\"message\":\"We have found your order for Add Chatbot pull request. What would you like to know?\"}}"
+    },
+    {
+      "project_id":"b3fpk8",
+      "trace_type":"text",
+      "trace_payload":"{\"payload\":{\"message\":\"It is expected to arrive on November 8th, at 10 am.\"}}"
+    },
+    {},
+    {
+      "project_id":"b3fpk8",
+      "trace_type":"text",
+      "trace_payload":"{\"payload\":{\"message\":\"Welcome to Commits 'R Us, the best toy store in town. How can I help you?\"}}"
+    },
+    {
+      "project_id":"b3fpk8",
+      "trace_type":"text",
+      "trace_payload":"{\"payload\":{\"message\":\"Let me track your order. What is your order number?\"}}"
+    },
+    {
+      "project_id":"b3fpk8",
+      "trace_type":"text",
+      "trace_payload":"{\"payload\":{\"message\":\"We have found your order for Add Chatbot pull request. What would you like to know?\"}}"
+    },
+    {
+      "project_id":"b3fpk8",
+      "trace_type":"text",
+      "trace_payload":"{\"payload\":{\"message\":\"It is expected to arrive on November 8th, at 10 am.\"}}"
+    },
+    {},
+    {
+      "project_id":"b3fpk8",
+      "trace_type":"text",
+      "trace_payload":"{\"payload\":{\"message\":\"Welcome to Commits 'R Us, the best toy store in town. How can I help you?\"}}"
+    },
+    {
+      "project_id":"b3fpk8",
+      "trace_type":"text",
+      "trace_payload":"{\"payload\":{\"message\":\"Let me track your order. What is your order number?\"}}"
+    },
+    {
+      "project_id":"b3fpk8",
+      "trace_type":"text",
+      "trace_payload":"{\"payload\":{\"message\":\"We have found your order for Add Chatbot pull request. What would you like to know?\"}}"
+    },
+    {
+      "project_id":"b3fpk8",
+      "trace_type":"text",
+      "trace_payload":"{\"payload\":{\"message\":\"Sorry we can't help you with that. Please contact us at 416-555-5555.\"}}"
```

```suggestion
      "trace_payload":"{\"payload\":{\"message\":\"Sorry we can't help you with that. Please contact us at 416-bes-tflo.\"}}"
```
Just a suggestion for all the phone number endings LOL